src/main/java/uk/ac/ebi/subs/fileupload/eventhandlers/PostFinishEvent.java

```diff
@@ -10,31 +11,50 @@
 import uk.ac.ebi.subs.fileupload.repository.util.FileHelper;
 import uk.ac.ebi.subs.fileupload.services.EventHandlerService;
 import uk.ac.ebi.subs.fileupload.util.FileStatus;
+import uk.ac.ebi.subs.fileupload.util.PropertiesLoader;
 
 import java.io.IOException;
+import java.nio.file.FileSystem;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.nio.file.StandardCopyOption;
+import java.util.Properties;
 
 /**
  * This class is handling the 'post-finish' hook event that is coming from the tusd server.
  * It is responsible to update the relevant existed file document in the MongoDB database.
  */
 public class PostFinishEvent implements TusEvent {
 
-    @Value("${file-upload.sourceBasePath}")
+    private static final Logger LOGGER = LoggerFactory.getLogger(PostFinishEvent.class);
+
+    private static final String APPLICATION_PROPERTIES_FILE = "application.yml";
```

Comment:
This looks odd - why do we need to specify the properties file name? Could we use the spring properties mechanism?